checkov/cloudformation/checks/resource/BaseCloudsplainingIAMCheck.py

```diff
@@ -13,7 +13,7 @@
 class BaseCloudsplainingIAMCheck(BaseResourceCheck):
     def __init__(self, name, id):
         # todo: managedpolicy
-        super().__init__(name=name, id=id, categories=CheckCategories.IAM, supported_resources=["AWS::IAM::Policy"])
+        super().__init__(name=name, id=id, categories=CheckCategories.IAM, supported_resources=["AWS::IAM::Policy", "AWS::IAM::ManagedPolicy"])
```

Comment:
@mikeurbanski1 FYI :)

Reply:
@njgibbon we can probably remove the todo from line 15 now

Reply:
Removed. I think that we will be able to extend to other ways of embedding policies e.g https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-iam-user.html#cfn-iam-user-policies but that requires a bit more thought because the fields are different and so you need to do different parsing to get to the policie(s).